test/spec/test-parallel-uploads.js

```diff
@@ -1,5 +1,3 @@
-'use strict'
```

Comment:
Do you know what the reasoning is behind this? Just curious

I thought you could remove for ts, if tsconfig had strict enabled, otherwise it still made sense, but I could be wrong.

Reply:
Biome recommended removing this because of its [`noRedundantUseStrict` rule](https://biomejs.dev/linter/rules/no-redundant-use-strict/) that says that `'use strict'` is not necessary for `.js` files, when the nearest `package.json` includes `type: module`, which is the case for tus-js-client. Therefore, I removed it. However, I am also not sure if the tools we use for building and bundling will still use strict mode for those files. But I am willing to test it since those are only the test files and thus not directly delivered to the package users.